hugegraph-api/src/main/java/com/baidu/hugegraph/core/GraphManager.java

```diff
@@ -1050,6 +1052,27 @@ private void serverStarted() {
         }
     }
 
+    private <T> void graphUpdateHandler(T response) {
```

Comment:
this method is just `graphModeUpdateHandler`, not `graphUpdateHandler`

Reply:
Not only for graphModeUpdateHandler, it could process other modify event